reddit_gold/public/static/css/gold.less

```diff
@@ -839,6 +839,17 @@ section#postcard {
 
     .username {
         font-size: 24px;
+        margin-bottom: 0.5em;
+    }
+
+    .gold-accent {
+        padding: 5px 10px;
+        border-radius: 4px;
+        margin-top: 0px;
+
+        a {
+            color: #9A7D2E;
+        }
```

Comment:
Tiny nit: The [style guide](https://github.com/reddit/styleguide/tree/master/css) says to indent by 2 spaces instead of 4.